main.scss

```diff
@@ -7,4 +9,23 @@
 
 	// Set to silent again to avoid being output twice
 	$o-comments-is-silent: true !global;
+
+	.o-comments__displayname-prompt {
```

Comment:
Instead of a custom class on the overlay, would setting the width of the overlay content `o-comments-displayname-form` do the trick?

Reply:
Yes it probably would, I can look into that. I added the custom class so I could nest this to keep it all isolated so then felt like adding the width on it as well did no harm but happy to look at improving it